### DISCUSSION TITLE
[dubbo][dubbo-spi-extensions] The CI environment in the extension library needs to support automatic testing of multiple Dubbo versions

### DISCUSSION THREAD

**Discussion by songxiaosheng** [apache/dubbo#13833] [General]
PR in Dubbo's main repository supports compatibility checks with the master branch of the associated extension library during CI environment construction

The CI environment in the extension library needs to support automatic testing of multiple Dubbo versions, which can ensure that the extension library supports multiple Dubbo versions and ensures forward compatibility

For example, testing multiple Dubbo versions in the case library：
<img width="306" alt="image" src="https://github.com/apache/dubbo/assets/81170548/8c56c541-1a20-4097-9fbe-17cc6cc7ecaa">

**Comment by AlbumenJ** [comment]
Track in https://github.com/apache/dubbo/issues/13865
